numpy/linalg/linalg.py

```diff
@@ -21,7 +21,7 @@
 
 from numpy.core import (
     array, asarray, zeros, empty, empty_like, intc, single, double,
-    csingle, cdouble, inexact, complexfloating, newaxis, all, Inf, dot,
+    float16, csingle, cdouble, inexact, complexfloating, newaxis, all, Inf, dot,
```

Comment:
Importing the alias `half` would be more consistent with the use of `double` and `single`